src/app/malaria-101/activity-1/activity-1-1.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="row">
+	<div class="col-md-4 col-md-offset-3">
+		<video id="video" width="640" height="480" src="../../assets/videos/malaria.mp4" controls autoplay></video>
+	</div>
+</div>
+<div class="row">
+    <button prev class="col-lg-5 col-md-offset-2">
```

Comment:
what's the point of this approach though? you aren't reducing the amount of duplicated code & to the contrary, you are complicating the entire implementation.

i was thinking about template directive. have a look at some of the directive examples [here](https://docs.angularjs.org/guide/directive).

so, imagine you have an element directive called `button-navigation` or something, which contains both of the buttons (previous & next) with styling. then, for all pages, you just need to call the directive: `<button-navigation></button-navigation>`, and everything'll auto-replaced by angular.